test/TestCommandsInsteadOfModulesRule.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=preferred-module  # FIXME: remove once migrated per GH-725
+"""command-instead-of-module rule testing."""
+import unittest
+
+from ansiblelint.rules import RulesCollection
+from ansiblelint.rules.CommandsInsteadOfModulesRule import CommandsInsteadOfModulesRule
+from ansiblelint.runner import Runner
+
+
+class TestCommandsInsteadOfModulesRule(unittest.TestCase):
+    """Unit test for command-instead-of-module rule."""
+
+    collection = RulesCollection()
+
+    def setUp(self):
+        """Register rule."""
+        self.collection.register(CommandsInsteadOfModulesRule())
+
+    def test_file_positive(self):
+        """Run tasks that should be successful."""
+        success = 'examples/playbooks/commands-instead-of-modules-success.yml'
+        good_runner = Runner(success, rules=self.collection)
+        self.assertEqual([], good_runner.run())
+
+    def test_file_negative(self):
+        """Run tasks that should fail."""
+        failure = 'examples/playbooks/commands-instead-of-modules-failure.yml'
+        bad_runner = Runner(failure, rules=self.collection)
+        errs = bad_runner.run()
+        self.assertEqual(3, len(errs))
```

Comment:
I would personally prefer to see the tests embedded directly into the rule file, like we already have with some of the rules, makes i much easier to debug the rule or make changes to it. Still, I will not use that as a blocker for your change.

Search for pytest inside existing rules and you will see how the newer tests looks like.

Reply:
Will check those out.

Reply:
> I would personally prefer to see the tests embedded directly into the rule file, like we already have with some of the rules, makes i much easier to debug the rule or make changes to it. Still, I will not use that as a blocker for your change.
> 
> Search for pytest inside existing rules and you will see how the newer tests looks like.

I just now realized that you probably wanted to get rid of `test/TestCommandsInsteadOfModulesRule.py` as well?

Reply:
If you can can yes. It makes not sense to have tests covering a rule in two places.